packages/cf-component-table/package.json

```diff
@@ -11,9 +11,13 @@
     "registry": "http://registry.npmjs.org/"
   },
   "dependencies": {
+    "cf-style-container": "^1.3.1",
     "prop-types": "^15.5.8"
   },
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0-0 || ^16.0.0-0"
+  },
+  "devDependencies": {
+    "cf-style-provider": "^2.0.0"
```

Comment:
There's no reason for devDeps since deving/testing is managed from the root level of cf-ui. Also, less dependencies = less version bumps by lerna.